bot/bot.go

```diff
@@ -80,6 +80,7 @@ func New() (*Borik, error) {
 	_ = parser.NewCommand("hooh", "Mirror the bottom half of an image.", MakeImageOpCommand(Hooh))
 	_ = parser.NewCommand("transform", "Apply transformations to an image.", MakeImageOpCommand(Transform))
 	_ = parser.NewCommand("invert", "Invert the colours of an image.", MakeImageOpCommand(Invert))
+	_ = parser.NewCommand("otsu", "Apply a threshold to an image using Otsu's method", MakeImageOpCommand(Otsu))
```

Comment:
```suggestion
	_ = parser.NewCommand("otsu", "Apply a threshold to an image using Otsu's method.", MakeImageOpCommand(Otsu))
```
All other command descriptions end with a period, one should be added here too.